### PR TITLE
Add PR comment commands for running instrumented tests and rebasing

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

File: .github/scripts/rebase_and_push.sh
Comment (vinaysshenoy) [inline .github/scripts/rebase_and_push.sh:4]: Where are these shall variables getting set from?

Comment (msasikanth) [reply .github/scripts/rebase_and_push.sh:4]: From the secrets, we already have these setup for `commit_and_push` & `amend_and_push`

Comment (msasikanth) [reply .github/scripts/rebase_and_push.sh:4]: So if you look at actual GH Action job, we are passing these variables from there

File: .github/workflows/ci_check_pr_comment.yml
Comment (Honey14) [inline .github/workflows/ci_check_pr_comment.yml:13]: So this will run when our url contains `/pull/` as well?

File: .github/workflows/ci_checks.yml
Comment (Honey14) [inline .github/workflows/ci_checks.yml:166]: We still want `qa-android-tests` as required, right? Why are we removing it from here?

**Review by Honey14 — COMMENTED** [review]
Looks good overall. Cannot see the instrumented tests running as github actions?

File: .github/workflows/ci_check_pr_comment.yml
Comment (msasikanth) [reply .github/workflows/ci_check_pr_comment.yml:13]: We are basically checking if the comment is for a pull request or not.

File: .github/workflows/ci_checks.yml
Comment (msasikanth) [reply .github/workflows/ci_checks.yml:166]: We are, if you look at `ci_checks_pr_comment.yml`, there is another `qa_purge_env` job that needs `qa_android_tests`. We removed it from here, because it's not longer part of this GH Action.

**Comment by msasikanth** [comment]
I think this PR is in a weird state now, where the `qa_android_tests` have been removed from the `ci_checks` file in this PR and so it cannot figure out a way to run that test yet. 🤔 

@vinaysshenoy I am marking the android tests as not required until we merge this in and re-enable it once the PR comment commands are available on `master`.

**Review by Honey14 — APPROVED** [review]
Looks good.

**Pull request merged**
